.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in the repo.
-*       @kostaspap @nhakmiller @austinbyers
+*       @kostaspap @nhakmiller @austinbyers @lindsey-w
```

Comment:
I've made a note to create official GitHub teams soon to match our company teams - then you can set the owner to your team!

Reply:
Awesome!